frontend/javascripts/oxalis/model/reducers/volumetracing_reducer_helpers.ts

```diff
@@ -179,6 +182,23 @@ export function setMappingNameReducer(
   if (mappingType !== "HDF5" || !isMappingEnabled) {
     mappingName = null;
   }
+  if (mappingName) {
+    const { agglomerates, mappings } = getSegmentationLayerByName(
+      state.dataset,
+      volumeTracing.tracingId,
+    );
+    const listToSearchIn = mappingType === "HDF5" ? agglomerates : mappings;
+    if (
+      mappingType === "HDF5" &&
+      (listToSearchIn == null || listToSearchIn.indexOf(mappingName) === -1)
+    ) {
+      console.error(
```

Comment:
If we want to notice when this happens because it indicates a bug, it should also be logged to airbrake